person.rb

```diff
@@ -0,0 +1,25 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..255)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def can_use_services?
+    return true if of_age? || @parent_permission
+
+    false
+  end
+
+  private
+
+  def of_age?
+    return true if @age >= 18
+
+    false
+  end
```

Comment:
- [OPTIONAL ] I would recommend refactoring the of_age? method to this to make it look more straightforward and simple. Like this :point_down: 

```
  def of_age?
      @age >= 18
  end
```

Reply:
Thank you very much @ridwanediallo for approving my pull request and giving insightful suggestions, I will definitely implement your suggestion 🍻